week-1-ts-ecosystem/w01e04-light-me-up/src/power-source.ts

```diff
@@ -0,0 +1,18 @@
+export class PowerSource {
+    private energySupply = 100
+
+    consume(energy: number): number {
+        if(this.energySupply - energy < 0){
+            throw new NoEnoughEnergyError();
+        } else {
```

Comment:
zapis `else` nie jest potrzebny bo `throw` nie pusci kodu dalej.

Wiec to co masz w środku `else` zostaw, ale bez zapisu z `else {`